### PR TITLE
Update documentation for Jib CLI war command 

### PR DESCRIPTION
**Pull request by mpeddada1**

### CODE REVIEW

File: docs/default_base_image.md [hunk 1]
```diff
@@ -81,4 +81,4 @@ For some reason if you have to keep the exact same behavior when using 3.0, you
 
 ## Jib CLI
 
-For the JAR mode, Jib CLI has always used AdoptOpenJDK.
+The JAR mode of the Jib CLI has always used AdoptOpenJDK and the WAR mode uses `jetty`.
```
Comment (chanseokoh) [inline docs/default_base_image.md:84]: Nice that you remembered this doc!

File: jib-cli/README.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ The CLI tool is powered by [Jib Core](https://github.com/GoogleContainerTools/ji
 * [Jar Command](#jar-command)
   * [Quickstart](#quickstart-1)
   * [Options](#options-1)
+* [War Command](#war-command)
+  * [Quickstart](#quickstart-2)
+  * [Options](#options-2)
+* [Options Shared Between Jar and War Command](#options-shared-between-jar-and-war-command)
```
Comment (chanseokoh) [inline jib-cli/README.md:40]:
```suggestion
* [Options Shared Between Jar and War Commands](#options-shared-between-jar-and-war-commands)
```

File: jib-cli/README.md [hunk 3]
```diff
@@ -156,6 +160,35 @@ jib jar --target <image name> path/to/myapp.jar [options]
...
+     $ docker run -p 8080:8080 cli-war-quickstart
+    ```
+## Options
+Flags for the `war` command:
+
+Option | Description
+---       | ---
+`--app-root` | The app root on the container. Customizing the app-root is helpful if you are using a different Servlet engine base image (for example, Tomcat)
+
+## Options Shared Between the Jar and War Command
```
Comment (chanseokoh) [inline jib-cli/README.md:189]:
```suggestion
## Options Shared Between the Jar and War Commands
```

**Review by chanseokoh — APPROVED** [review]

File: jib-cli/README.md (hunk 2 above)
Comment (mpeddada1) [reply jib-cli/README.md]: oops, thank you!

**Pull request merged**